webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala

```diff
@@ -222,10 +222,12 @@ class NodeGroupForm(
       & "group-save" #> { if (CurrentUser.checkRights(AuthorizationType.Group.Edit))
                     <span class="save-tooltip-container">
                       {
-                        SHtml.ajaxButton(<span>Save <i class="fa fa-download"></i></span>, onSubmit _)  %  ("id" -> saveButtonId) % ("class" -> " btn btn-success btn-icon")
+                          <button class="btn btn-success btn-icon ui-button ui-corner-all ui-widget" id="groupSaveButtonId"> <span>Save <i class="fa fa-download"></i></span></button>
+                          SHtml.ajaxOnSubmit(onSubmit _)
```

Comment:
I don't understand how `ajaxOnSubmit` works, I get this in UI :
![image](https://user-images.githubusercontent.com/23410978/190964259-ad4a5e8c-d334-43d2-b39b-98f1d0818fc0.png)